cyhy/mailer/cli.py

```diff
@@ -793,9 +797,13 @@ def send_cyhy_reports(
             if not to_emails:
                 continue
 
-            # Grab the email address of the appropriate CSA.
+            # Grab the email address of the appropriate CSA for non-Federal and
+            # non-International entities.
```

Comment:
This matches the case used everywhere else.

```suggestion
            # Grab the email address of the appropriate CSA for non-federal and
            # non-international entities.
```